lib/fastlane/actions/xcodebuild.rb

```diff
@@ -85,9 +92,6 @@ def self.run(params)
             # If not passed, retrieve path from previous xcodebuild call
             params[:archive_path] ||= Actions.lane_context[SharedValues::XCODEBUILD_ARCHIVE]
 
-            # Default to ipa as export format
-            params[:export_format] ||= "ipa"
```

Comment:
We cannot use `-exportOptionsPlist` and `-exportFormat` at the same time. Xcodebuild will throw an error:
```
xcodebuild: error: The flag -exportFormat cannot be specified along with -exportOptionsPlist
```

Users can still use `:export_format` but they have to use it explicitly.